### ISSUE TITLE
[go, postgres] relation not being found anymore

### ISSUE THREAD

**Issue by Streppel** [kyleconroy/sqlc#1078]
**schema.sql**

```sql
create table events
(
    ID int
);

create table handled_events
(
    last_handled_id int,
    handler text
);
```

**queries.sql**
```sql
-- name: SelectAllJoined :many
select e.* from events e
    inner join handled_events he
       on e.ID > he.last_handled_id
where he.handler = $1
    for update of he skip locked;
```

Before [commit 070717](https://github.com/kyleconroy/sqlc/commit/07071774fab4c9960feab2898678241ff233ea8a), this would generate the following query:

```go
const selectAllJoined = `-- name: SelectAllJoined :many
select e.ID from events e
    inner join handled_events he
       on e.ID > he.last_handled_id
where he.handler = $1
    for update of he skip locked
`
```

Now, an error is thrown

```❯ sqlc generate
# package db
queries.sql:1:1: relation "he" does not exist
```

The error goes away if we ditch the `for update` lock in the query

**Comment by kyleconroy** [comment]
What version of sqlc are you using?

**Comment by Streppel** [comment]
Hey @kyleconroy! Sorry for taking that long to reply. `sqlc version` outputs `v1.8.1-devel`. Building main branch locally and moving it to go/bin.

**Comment by kyleconroy** [comment]
I suggest downgrading to `v1.8.0`. The bug exists on main which hasn't been released. I'll make sure this is fixed before releasing `v1.9.0`

**Comment by Streppel** [comment]
thanks @kyleconroy!
